app/reducers/edit-entry.js

```diff
@@ -5,19 +5,19 @@ const defaultState = {
 
 export default function editEntry(state = defaultState, action) {
     switch (action.type) {
-        case 'UPDATE_USER':
-        case 'EDIT_ENTRY':
-            return {
-                editing: true,
-                underEdit: action.id
-            };
-        case 'SAVE_ENTRY': {
-            return {
-                editing: false,
-                underEdit: ''
-            };
-        }
-        default:
-            return state;
+    case 'UPDATE_USER':
```

Comment:
Corrected. ESLint thrown an error and I set loose the auto-fixing option. Now the .eslintrc has a fix for the switch statements.